python/python3-enzyme/python3-enzyme.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="python3-enzyme"
-VERSION="0.4.1"
+VERSION="0.5.1"
 HOMEPAGE="https://github.com/Diaoul/enzyme/"
-DOWNLOAD="https://github.com/Diaoul/enzyme/archive/refs/tags/0.4.1/enzyme-0.4.1.tar.gz"
-MD5SUM="eb2c076cbd784007b4a3f880fe1590f1"
+DOWNLOAD="https://github.com/Diaoul/enzyme/archive/refs/tags/0.5.1/enzyme-0.5.1.tar.gz"
+MD5SUM="a3a9abfc7e590b10221b800996b90644"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
 REQUIRES=""
```

Comment:
you need python3-wheel and python3-build for this

Reply:
Oops, I caught this after I committed the change and reverted the commit locally and created a new one, but I forgot to push it to GitHub.